src/ui/zcl_abapgit_services_abapgit.clas.abap

```diff
@@ -5,13 +5,13 @@ CLASS zcl_abapgit_services_abapgit DEFINITION
 
   PUBLIC SECTION.
 
-    CONSTANTS: c_abapgit_repo     TYPE string   VALUE 'https://github.com/abapGit/abapGit'     ##NO_TEXT,
-               c_abapgit_homepage TYPE string   VALUE 'https://www.abapgit.org'                ##NO_TEXT,
-               c_abapgit_wikipage TYPE string   VALUE 'https://docs.abapgit.org'               ##NO_TEXT,
-               c_dotabap_homepage TYPE string   VALUE 'https://dotabap.org'               ##NO_TEXT,
-               c_abapgit_package  TYPE devclass VALUE '$ABAPGIT'                              ##NO_TEXT,
-               c_abapgit_url      TYPE string   VALUE 'https://github.com/abapGit/abapGit.git' ##NO_TEXT,
-               c_abapgit_class    TYPE tcode    VALUE `ZCL_ABAPGIT_REPO`                      ##NO_TEXT.
+    CONSTANTS c_abapgit_repo TYPE string VALUE 'https://github.com/abapGit/abapGit' ##NO_TEXT.
+    CONSTANTS c_abapgit_homepage TYPE string VALUE 'https://www.abapgit.org' ##NO_TEXT.
+    CONSTANTS c_abapgit_wikipage TYPE string VALUE 'https://docs.abapgit.org' ##NO_TEXT.
+    CONSTANTS c_dotabap_homepage TYPE string VALUE 'https://dotabap.org' ##NO_TEXT.
+    CONSTANTS c_abapgit_package TYPE devclass VALUE '$ABAPGIT' ##NO_TEXT.
+    CONSTANTS c_abapgit_url TYPE string VALUE 'https://github.com/abapGit/abapGit.git' ##NO_TEXT.
+    CONSTANTS c_abapgit_class TYPE seoclsname VALUE `ZCX_ABAPGIT_EXCEPTION` ##NO_TEXT.
```

Comment:
Should probably remove the pragmas, while you're "already here" (see #3827 or #3555). 
@larshp Is it intentional that this is not caught by the abaplint rule `forbidden_pseudo_and_pragma`? Can open an issue otherwise

Reply:
Never mind, looks like the toggle `ignoreGlobalClassDefinition` is set. Any specific reason that that toggle is `true`?

Reply:
SE80 will add the NO_TEXT pragmas when editing the class